blog/MeetthehoodiesNickHehr.md

```diff
@@ -0,0 +1,33 @@
+---
+
+title: 'Meet the Hoodies: Nick Hehr'
+author: Pawan
+layout: blog-post
+comments: true
+
+---
+
+#Hello all! I recently had the great honor of interviewing Nick Hehr (@hipsterbrown on twitter and github). Let's see what he had to say:
```

Comment:
Maybe instead of `(@hipsterbrown on twitter and github)`, we could add links to Nick’s accounts with `([Twitter](https://twitter.com/hipsterbrown), [GitHub](https://github.com/hipsterbrown))`?